packages/node_modules/@node-red/editor-client/src/js/ui/palette-editor.js

```diff
@@ -882,7 +882,7 @@ RED.palette.editor = (function() {
 
         if (RED.settings.theme('palette.upload') !== false) {
             var uploadSpan = $('<span class="button-group">').prependTo(toolBar);
-            var uploadButton = $('<button type="button" class="red-ui-sidebar-header-button red-ui-palette-editor-upload-button"><label><i class="fa fa-upload"></i><form id="red-ui-palette-editor-upload-form" enctype="multipart/form-data"><input name="tarball" type="file" accept=".tgz"></label></button>').appendTo(uploadSpan);
+            var uploadButton = $('<button type="button" class="red-ui-sidebar-header-button red-ui-palette-editor-upload-button"><label><i class="fa fa-upload"></i><form id="red-ui-palette-editor-upload-form" enctype="multipart/form-data" style="width: 0px;"><input name="tarball" type="file" accept=".tgz"></label></button>').appendTo(uploadSpan);
```

Comment:
I would prefer we add CSS to the stylesheet and not inline it here.

https://github.com/node-red/node-red/blob/1edf5acb8757965a89167376840304e9d6991662/packages/node_modules/%40node-red/editor-client/src/sass/palette-editor.scss#L240

Reply:
Thank you for the comment.
I've moved the `width` specification to CSS.